<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="yt.lan" name="com.yinting.core.test1" tests="4" failures="0" timestamp="07 六月 2017 13:29:06 GMT" time="0.000" errors="4">
  <testcase name="testd" time="0.000" classname="com.yinting.core.test1">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: testd([])
Arguments: [(java.lang.Integer)0]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: testd([])
Arguments: [(java.lang.Integer)0]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:49)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1187)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- testd -->
  <testcase name="testc" time="0.000" classname="com.yinting.core.test1">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: testc([])
Arguments: [(java.lang.Integer)0]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: testc([])
Arguments: [(java.lang.Integer)0]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:49)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1187)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- testc -->
  <testcase name="testa" time="0.000" classname="com.yinting.core.test1">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: testa([])
Arguments: [(java.lang.Integer)0]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: testa([])
Arguments: [(java.lang.Integer)0]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:49)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1187)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- testa -->
  <testcase name="testb" time="0.000" classname="com.yinting.core.test1">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: testb([])
Arguments: [(java.lang.Integer)0]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: testb([])
Arguments: [(java.lang.Integer)0]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:49)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1187)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- testb -->
</testsuite> <!-- com.yinting.core.test1 -->
